components/infobox/wikis/starcraft/infobox_league_custom.lua

```diff
@@ -192,19 +192,19 @@ function CustomLeague._playerRaceBreakDown()
 		if zergNumber + terranNumbner + protossNumber + randomNumber > 0 then
 			playerBreakDown.display = {}
 			if protossNumber > 0 then
-				table.insert(playerBreakDown.display, RaceIcon.getSmallIcon({'p'})
+				table.insert(playerBreakDown.display, Faction.Icon{size = 'small', showLink = false, showTitle = false, faction = 'p'}
```

Comment:
`size = 'small', showLink = false` are defaults (unless you want it to be explicit)

Reply:
removing title is intended? it was present with `RaceIcon.getSmallIcon`